roles/influxdb_exporter/molecule/alternative/molecule.yml

```
@@ -0,0 +1,10 @@
+---
+provisioner:
```

Please include the shared prepare step here

```suggestion
provisioner:
  playbooks:
    prepare: "${MOLECULE_PROJECT_DIRECTORY}/../../.config/molecule/alternative/prepare.yml"
```